test/util/test_runners/test_runner_args.py

```diff
@@ -0,0 +1,44 @@
+from argparse import ArgumentParser
+
+
+class TestRunnerArgs:
+    """
+    Dto to hold the tests environment arguments
+    """
+
+    def __init__(self, hostname, account, username, password, invoke_cli_as_process, working_cli_path_only_for_init: str,
```

Comment:
should these also have defaults? `hostname=https://...` so that we have a default flow?

Reply:
no this should shout if param is missing